### PR TITLE
[script.wellbeing] 1.0.0

### PR DESCRIPTION
**Pull request by Heckie75**
### Description
Kodi addon in order to limit playback time per day, e.g. for parental control

Features:
- Limit playback time per day, e.g. 3 hours video playback
- Set rest periods per day, e.g. Monday night from 22:00 to 07:00

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

### CODE REVIEW

File: script.wellbeing/addon.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	<extension point="xbmc.service" library="addon.py" />
+	<extension point="xbmc.python.script" library="script.py" />
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en_GB">Wellbeing</summary>
+		<summary lang="de_DE">Wellbeing</summary>
+		<description lang="en_GB">Addon in order to limit playback time per day, e.g. for parental control. After playback time has been reached playback will be stopped automatically.
+		</description>
+		<description lang="de_DE">Addon um Wiedergabezeit pro Tag zu begrenzen. Nachdem die Wiederabezeit erreicht ist, beendet das Addon die Wiedergabe automatisch.
+		</description>
+		<language>en_GB</language>
```
Comment (basrieter) [inline script.wellbeing/addon.xml:15]: I think this should be "en"

File: script.wellbeing/addon.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		<language>de_DE</language>
```
Comment (basrieter) [inline script.wellbeing/addon.xml:16]: And this "de"

File: script.wellbeing/addon.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.wellbeing" name="Wellbeing" version="1.0.0" provider-name="Heckie">
+	<requires>
+		<import addon="xbmc.python" version="3.0.0" />
+	</requires>
+	<extension point="xbmc.service" library="addon.py" />
+	<extension point="xbmc.python.script" library="script.py" />
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en_GB">Wellbeing</summary>
```
Comment (basrieter) [inline script.wellbeing/addon.xml:9]: Make my brain happy with a fix in indentation here :)

**Review by basrieter — CHANGES_REQUESTED** [review]
Please see my  minor comments.

File: script.wellbeing/addon.xml (hunk 1 above)
Comment (Heckie75) [reply script.wellbeing/addon.xml:15]: ok

File: script.wellbeing/addon.xml (hunk 2 above)
Comment (Heckie75) [reply script.wellbeing/addon.xml:16]: ok

File: script.wellbeing/addon.xml (hunk 3 above)
Comment (Heckie75) [reply script.wellbeing/addon.xml:9]: I formatted it again but I can't see any difference. What is wrong with the indentation?

**Review by basrieter — APPROVED** [review]

**Comment by basrieter** [comment]
Thanks!

**Pull request merged**